flutter_lyra/example/ios/Podfile

```diff
@@ -37,5 +37,14 @@ end
 post_install do |installer|
   installer.pods_project.targets.each do |target|
     flutter_additional_ios_build_settings(target)
+    # TODO: Remove this when Cocoapods 13.0 is released
```

Comment:
Do you mean `v1.13` ?
If so, it's released: https://rubygems.org/gems/cocoapods/versions?locale=fr